en/docs/ai-workspace/1.0.0/policies/build-gateway-with-ai-policies.md

```
@@ -0,0 +1,193 @@
+| Field | Description |
+|---|---|
+| `gomodule` | The Go module reference for a Policy Hub managed policy, for example `github.com/wso2/gateway-controllers/policies/pii-masking@v1` |
+| `filePath` | The path from `build.yaml` to a local custom AI policy directory |
+
+### Add a custom AI policy
+
+Your custom AI policy can reside anywhere on the filesystem. Use a relative path from the `build.yaml` file to point to it.
+
+For example, if your directory layout is:
+
+```text
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)